code-reviews/recipes/Go.md

```diff
@@ -30,15 +36,87 @@ One awesome feature of `golangci-lint` is that is can be easily introduced to an
 
 `gofmt` should be run as a part of every build to enforce the common standard.
 
-To automate this process in Azure Devops you can add the following snippet to you `azure-pipelines.yaml` file. This will lint any scripts in the `./scripts/` folder.
+To automate this process in Azure Devops you can add the following snippet to you `azure-pipelines.yaml` file. This will format any scripts in the `./scripts/` folder.
 
-> TODO: Add example of yaml for gofmt
+`- script: go fmt`\
+  `workingDirectory: $(System.DefaultWorkingDirectory)/scripts`\
+  `displayName: "Run code formatting"`
+  
+  `govet` should be run as a part of every build to check code linting.
+
+To automate this process in Azure Devops you can add the following snippet to you `azure-pipelines.yaml` file. This will check linting of any scripts in the `./scripts/` folder.
+
+`- script: go vet`\
+  `workingDirectory: $(System.DefaultWorkingDirectory)/scripts`\
+  `displayName: "Run code linting"`
 
 ## Pre-Commit Hooks
 
 All developers should run `gofmt` in a pre-commit hook to ensure standard formatting.
 
-> TODO: Add example of how this is installed and used
+### Step 1- Install pre-commit
```

Comment:
Since the guidance above encourages the use of `vscode-go`, the IDE can be configured to run `gofmt` using the "Format on Save" setting. We should consider having that guidance here in addition to the pre-commit hook.

Reply:
https://github.com/microsoft/vscode-go/wiki/On-Save-features
While using vscode-go, fmt and vet are run by default in vscode on every file save.
Should we add how to remove the default setting?

Reply:
Ah if it's the default, I wouldn't worry about it. It has been ages since I setup vscode-go as I've been using it non-stop since its release :) so I didn't recall what was the default enabled.

Reply:
Thanks for your good feedbacks though :). Resolving this comment